### DISCUSSION TITLE
Group Assignment #3 📋

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/24SU-ASIAAM-191A#43] [Group Assignment]
### Due 7/17

Feel free to post your revised memo here or email it to me when you would like me to review it.

### Please submit your surveys via this link (NOT THIS THREAD!):

- https://forms.gle/Cdr29ZmYt3qXnvRMA

**Comment by stinest** [comment]
Here is our group's revised memo! (Group 4 - Community on Campus: Palestinian Solidarity at UCLA): https://docs.google.com/document/d/1Pvp0Lycq1uSwPRjI9X7VpGJz8C_6isw6jjHTKnTOQ3E/edit?usp=sharing
